src/apps/dashboard/controllers/networking.js

```diff
@@ -55,6 +55,8 @@ function onSubmit(e) {
                 config.RequireHttps = form.querySelector('#chkRequireHttps').checked;
                 config.BaseUrl = form.querySelector('#txtBaseUrl').value;
                 config.EnableRemoteAccess = form.querySelector('#chkRemoteAccess').checked;
+                config.PublicUserListing = form.querySelector('#selectPublicUserListing').value === 'always' || form.querySelector('#selectPublicUserListing').value === 'local';
+                config.PublicUserListingLocalOnly = form.querySelector('#selectPublicUserListing').value === 'local';
```

Comment:
Maybe take the value once?

Reply:
As in retrieve `form.querySelector('#selectPublicUserListing').value` into a variable and compare against that?

Reply:
Yes